### PR TITLE
fix: args parse issue

### PR DESCRIPTION
**Pull request by yxxhero**
Signed-off-by: yxxhero <aiopsclub@163.com>

fix: args parse issue

### CODE REVIEW

**Comment by yxxhero** [comment]
stringlist not only `-f` `--set` `--values`

**Comment by yxxhero** [comment]
@mumoshu @itscaro

**Comment by mumoshu** [comment]
@yxxhero Hey! Is this a fix for a new issue introduced via the migration to cobra? Or, has this been there for a longer period?

**Comment by yxxhero** [comment]
this been there for a longer period.

**Comment by yxxhero** [comment]
@mumoshu

**Comment by yxxhero** [comment]
@itscaro

File: pkg/argparser/args.go [hunk 1]
```diff
@@ -38,54 +54,44 @@ func newArgMap() *argMap {
...
-				} else {
-					// check if next value is arg to flag
-					if index+1 < len(argsVals) {
-						nextVal := argsVals[index+1]
-						if strings.HasPrefix(nextVal, "--") {
-							argsMap.SetArg(arg, "", false)
-						} else {
-							argsMap.SetArg(arg, nextVal, true)
-						}
+func argsAnalysis(am *argMap, args string) {
```
Comment (itscaro) [inline pkg/argparser/args.go:57]:
```suggestion
func analyzeArgs(am *argMap, args string) {
```

I prefer using verb

Comment (yxxhero) [reply pkg/argparser/args.go:57]: ok.

**Comment by yxxhero** [comment]
@itscaro done. Thanks very much.

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks for fixing it!

**Pull request merged**